packages/indexer/README.md

```diff
@@ -79,6 +79,30 @@ for await (const tx of txCollector.collect()) {
 
 It will fetch transactions between `[fromBlock, toBlock]`, which means both `fromBlock` and `toBlock` are included in query range.
 
+Fine grained query for transactions by scripts with `io_type` is supported: 
+
+```javascript
+txCollector = new TransactionCollector(indexer, {
+  lock: {
+    script: {
+      code_hash: 
+        "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8",
+      hash_type: "type",
+      args: "0xa528f2b9a51118b193178db4cf2f3db92e7df323",
+    },
+    io_type: "input",
```

Comment:
Here we are still using snake case.

Reply:
Sorry for missing these `io_type`. Will fix it.